src/app/Fake.DotNet.Paket/Paket.fs

```diff
@@ -144,6 +144,7 @@ let internal createProcess (runType:StartType) =
         |> Arguments.appendIf parameters.IncludeReferencedProjects "--include-referenced-projects"
         |> List.foldBack (fun t -> Arguments.append ["--exclude"; t]) parameters.ExcludedTemplates
         |> List.foldBack (fun (id, v) -> Arguments.append ["--specific-version"; id; v]) parameters.SpecificVersions
+        |> Arguments.append [parameters.OutputPath]
```

Comment:
Is it ok to put it at the end? (I don't recall what append does but I think it adds to the end)

Reply:
I also hope that it is added to the end =)
I checked docs https://fsprojects.github.io/Paket/paket-pack.html
and find  out that it should last param
<img width="892" alt="paket_pack" src="https://user-images.githubusercontent.com/1197905/67518827-37a7d880-f6ae-11e9-8274-a9d0ea72948a.png">